portfolio/src/main/webapp/index.html

```diff
@@ -39,6 +39,15 @@ <h3>Some of her favorite moments!</h3>
               <li>Meeting fellow CS interns over community chats</li>
             </ul>
           <h3>Songs that she's currently listening to!</h3>
+            
+            <div> 
+              <button class="song-buttons" id="prev" onclick="changeImg(true)"><</button>
```

Comment:
don't set the onclick handler here; let's add it in the js so you can get rid of the global vars.

Reply:
Does global vars refer to having index variable to keep track of which image we are on, regarding the slideshow? Thank you :)